File: contest/atcoder/config/custom_template.py
#!/usr/bin/env python3
import sys
from collections.abc import Iterable
from math import *
from itertools import *
from collections import *
from functools import *
from operator import *
try:
    from math import gcd
except Exception:
    from fractions import gcd
{% if prediction_success %}
{% else %}
readInt = lambda: int(sys.stdin.readline())
readIntN = lambda: [int(v) for v in sys.stdin.readline().split(' ')]
{% endif %}

{% if mod %}
MOD = {{ mod }}  # type: int
{% endif %}
{% if yes_str %}
YES = "{{ yes_str }}"  # type: str
{% endif %}
{% if no_str %}
NO = "{{ no_str }}"  # type: str
{% endif %}

{% if prediction_success %}
def solve({{ formal_arguments }}):
    return 0

{% endif %}

def main():
    {% if prediction_success %}
    def iterate_tokens():
        for line in sys.stdin:
            for word in line.split():
                yield word
    tokens = iterate_tokens()
    {{ input_part }}
    result = solve({{ actual_arguments }})
    if isinstance(result, Iterable):
        result = '\n'.join([str(v) for v in result])
    {% else %}
    # Failed to predict input format
    result = 0
    {% endif %}
    print(result)

if __name__ == '__main__':
    main()

File: contest/atcoder/config/custom_template.cpp
#include <bits/stdc++.h>
using namespace std;

{% if mod %}
const long long MOD = {{ mod }};
{% endif %}
{% if yes_str %}
const string YES = "{{ yes_str }}";
{% endif %}
{% if no_str %}
const string NO = "{{ no_str }}";
{% endif %}

{% if prediction_success %}
auto solve({{ formal_arguments }}){

}
{% endif %}

int main(){
    {% if prediction_success %}
    {{input_part}}
    auto result = solve({{ actual_arguments }});
    {% else %}
    auto result = 0
    // Failed to predict input format
    {% endif %}
    cout << result << endl;
    return 0;
}
